generate.js

```diff
@@ -18,10 +18,10 @@ function normalizeSlices(rawSlices) {
     .filter(x => x !== '');
 }
 
-async function generatePayload(rawSlices) {
+async function generatePayload(rawSlices, customSha = null) {
   const {owner, repo} = github.context.repo;
   const branch = github.context.ref.trim().replace('refs/heads/', '');
```

Comment:
@driv3r can we here also have the possibility of a custom branch ref? Because I assume if we are running a scheduled job in the main/master it will always show main branch form the context, which would be a bit confusing in the deploy app build's list.